class InternalException(Exception):
    pass


class EntityNotFoundException(Exception):
    pass
